package.json

```diff
@@ -18,13 +19,13 @@
     "vitest": "^3.1.3"
   },
   "scripts": {
-    "build": "esbuild src/index.ts --bundle --platform=node --target=node20 --minify --outfile=dist/index.js",
+    "build": "esbuild src/index.ts --bundle --platform=node --target=node20 --format=esm --minify --outfile=dist/index.js",
```

Comment:
In theory... this should be quite safe but then... who knows. Perhaps this should only land after testing this e2e in some repo. It feels like a somewhat risky change.

Reply:
Damn. That's not great :p I think I'll still try to run with it - but I have to definitely test it e2e first